src/main/java/com/example/gildedrose/item/BackstagePassesItem.java

```diff
@@ -1,4 +1,8 @@
-public class BackstagePassesItem implements Itemable{
+package com.example.gildedrose.item;
+
+import com.example.gildedrose.item.Itemable;
+
+public class BackstagePassesItem implements Itemable {
```

Comment:
Naming is always a challenge.  I thought of Itemable as something that could be itemized.  We can rename.

Reply:
Definitely. I would think that `Itemizable` would be an appropriate role name for that action.  But then I would also expect a method named `itemize` instead of `update`